build.rs

```diff
@@ -145,7 +145,10 @@ fn cross_compiling() -> Result<Option<CrossCompileConfig>> {
     let target_vendor = cargo_env_var("CARGO_CFG_TARGET_VENDOR");
     let target_os = cargo_env_var("CARGO_CFG_TARGET_OS");
 
-    if cross_lib_dir.is_none() && cross_python_version.is_none() {
+    if env_var("PYO3_CROSS").is_none()
+        && cross_python_version.is_none()
```

Comment:
I think this should be `cross_lib_dir.is_none()`.

Reply:
👍 good catch, thanks!